index.bs

```diff
@@ -241,12 +241,34 @@ OBUs stored in the configOBUs field follow the [=open_bitstream_unit=] [=Low Ove
 The <dfn>initial_presentation_delay_present</dfn> field indicates the presence of the initial_presentation_delay_minus_one field.
 
 The <dfn>initial_presentation_delay_minus_one</dfn> field indicates the number of samples (minus one) that need be decoded prior to starting the presentation of the first sample associated with this sample entry in order to guarantee that each sample will be decoded prior to its presentation time under the constraints of the first level value indicated by [=seq_level_idx=] in the [=Sequence Header OBU=] (in the configOBUs field or in the associated samples). More precisely, the following procedure SHALL not return any error:
```

Comment:
I changed the ordering here to match that of the record in PR57. I think it's clearer if they match, but not the end of the world if they do not.

Reply:
I don't understand.

Reply:
My fault. I was looking at the wrong diff, or was confused by an earlier diff. Sorry!